xchange-core/src/main/java/org/knowm/xchange/dto/trade/OpenPosition.java

```diff
@@ -0,0 +1,110 @@
+package org.knowm.xchange.dto.trade;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.Set;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.Order;
+
+public class OpenPosition extends Order {
+
+  public OpenPosition(
+      OrderType type,
+      BigDecimal originalAmount,
+      CurrencyPair currencyPair,
```

Comment:
CurrencyPair is not good enough for contracts, see: https://github.com/knowm/XChange/issues/3171#issuecomment-595261528

Reply:
Sure, but we don't have a new DTO in order to support that. That is why i will put CurrencyPair for the near future